app/controllers/api/v1/messages_controller.rb

```diff
@@ -2,19 +2,21 @@ module Api
   module V1
     class MessagesController < Api::V1::ApiController
       def index
-        @messages = policy_scope(conversation.messages).includes(:user)
+        conversation = find_conversation
+        @messages = policy_scope(conversation.messages)
+        render_messages(@messages)
       end
 
       def create
-        @conversation = conversation
-        @message = build_message
+        conversation = find_conversation
+        message = build_message(conversation)
 
-        authorize @message
+        authorize message
 
-        if save_message
-          render json: { message: { content: @message.content } }, status: :created
+        if save_message(message)
+          render_created_message(message)
```

Comment:
The same here, I think `render :create` is enough, this call the view and return the data and the status

Reply:
Done!